ASP.NET Core_EF_CodeFirst/Controllers/StudentController.cs

```diff
@@ -14,24 +15,28 @@ public StudentController(IStudentService studentService)
             this.studentService = studentService;
         }
 
+        [Authorize]
```

Comment:
Please use Authorized controller with this attribute and don't use the attribute for methods.